lib/ansible/modules/cloud/amazon/aws_polly_speech.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2018, Aaron Smith <ajsmith10381@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+module: aws_polly_speech
+short_description: Synthesize speech using AWS Polly.
+description:
+    - Synthesizes UTF-8 input, plain text or SSML, to a stream of bytes. SSML input must be valid, well-formed SSML.
+author: "Aaron Smith (@slapula)"
+version_added: "2.7"
+requirements: [ 'botocore', 'boto3' ]
+options:
+  lexicons:
+    description:
+    - List of one or more pronunciation lexicon names you want the service to apply during synthesis.
+  output_format:
+    description:
+    - The format in which the returned output will be encoded.
+    choices: ['json', 'mp3', 'ogg_vorbis', 'pcm']
+    required: true
+  sample_rate:
+    description:
+    - The audio frequency specified in Hz.
+    default: '16000'
+  speech_mark_types:
+    description:
+    - The type of speech marks returned for the input text.
+    choices: ['sentence', 'ssml', 'viseme', 'word']
+  text_type:
+    description:
+    - Specifies whether the input text is plain text or SSML.
+    choices: ['ssml', 'text']
+    default: 'text'
+  voice_id:
+    description:
+    - Voice ID to use for the synthesis.
+    choices: ['Geraint', 'Gwyneth', 'Mads', 'Naja', 'Hans', 'Marlene', 'Nicole', 'Russell', 'Amy', 'Brian', 'Emma', 'Raveena', 'Ivy', 'Joanna',
+        'Joey', 'Justin', 'Kendra', 'Kimberly', 'Matthew', 'Salli', 'Conchita', 'Enrique', 'Miguel', 'Penelope', 'Chantal', 'Celine', 'Mathieu',
+        'Dora', 'Karl', 'Carla', 'Giorgio', 'Mizuki', 'Liv', 'Lotte', 'Ruben', 'Ewa', 'Jacek', 'Jan', 'Maja', 'Ricardo', 'Vitoria', 'Cristiano',
+        'Ines', 'Carmen', 'Maxim', 'Tatyana', 'Astrid', 'Filiz', 'Vicki', 'Takumi', 'Seoyeon', 'Aditi']
```

Comment:
This seems like something that will change frequently, is there an API method to list these so we can do validation at run-time and not stop people from using new voices as they become available.